src/lib/keyseq.ts

```diff
@@ -52,12 +55,23 @@ export class MinimalKey {
     }
 
     /** Does this key match a given MinimalKey extending object? */
-    public match(keyevent) {
+    // NB: not symmetric!
+    public match(keyevent): true | false | "skip" {
         // 'in' doesn't include prototypes, so it's safe for this object.
         for (const attr in this) {
             // Don't check shiftKey for normal keys.
             if (attr === "shiftKey" && this.key.length === 1) continue
-            if (this[attr] !== keyevent[attr]) return false
+            if (attr === "keydown") {
```

Comment:
To avoid editing the real event we can add keyup handling here in a similar fashion